deploy/support/index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Support and Feedback
+---
+
+If you have any questions or feedback about Deno Deploy, please reach out to us
+on the [Deno Discord](https://discord.gg/deno) in the `#deploy-ea` channel or
```

Comment:
Let's preempt the merging of the discord channels and just advertise the `deploy` channel

```suggestion
on the [Deno Discord](https://discord.gg/deno) in the `#deploy` channel or
```